federation/ldap/src/main/java/org/keycloak/storage/ldap/mappers/membership/UserRolesRetrieveStrategy.java

```diff
@@ -62,7 +62,8 @@ public List<LDAPObject> getLDAPRoleMappings(CommonLDAPGroupMapper roleOrGroupMap
 
                 Condition membershipCondition = getMembershipCondition(membershipAttr, userMembership);
                 ldapQuery.addWhereCondition(membershipCondition);
-                return ldapQuery.getResultList();
+
+                return ldapUtils.loadAllLDAPObjects(ldapQuery, ldapConfig);
```

Comment:
There's a typo here leading to compilation errors. Should be `LDAPUtils` instead of `ldapUtils`

Reply:
Absolutely right, forgot to push the fix when I compiled and it failed. Done.